src/main/java/frc/robot/Robot.java

```diff
@@ -278,20 +279,27 @@ public CommandBase visionAlignment() {
      */
     private void configureButtonBindings() {
         driveController.getButton(Button.kA).whenHeld(intake.intake());
-        driveController.getButton(Button.kB).whileHeld(shootNBalls(5)).whenReleased(shooter.spinDown());
-        driveController.getButton(Button.kX).whileHeld(maxSpeedNBalls()).whenReleased(shooter.spinDown());
+        driveController.getButton(Button.kB).whileHeld(shootNBalls(5)).whenReleased(shooter.SlowSpin());
+        driveController.getButton(Button.kX).whileHeld(maxSpeedNBalls()).whenReleased(shooter.SlowSpin());
         driveController.getButton(Button.kY).toggleWhenActive(
                 drive.drive(() -> -driveController.getTriggers(), () -> driveController.getX(Hand.kLeft)));
         driveController.getButton(Button.kBack).whenPressed(cancelAll());
         driveController.getButton(Button.kStart).whenPressed(visionAlignment());
         driveController.getButton(Button.kBumperRight).whenHeld(drive.slowTurn(true));
         driveController.getButton(Button.kBumperLeft).whenHeld(drive.slowTurn(false));
-
+        final DpadTrigger stopShooter = DpadTrigger.DpadDown(driveController);
```

Comment:
Why make the triggers just to use them once in the next line? Combine them please